src/App.js

```diff
@@ -1,11 +1,17 @@
 import "./App.css";
+import Footer from "./Components/Home/Footer";
+import Header from "./Components/Home/Header";
+import Landing from "./Components/Home/Landing";
 
 function App() {
+  
   return (
     <div className="App">
-      
+      <Header />
```

Comment:
you can install `react-router-dom` soon and after every page you make, just add the route

Reply:
will be change them after all the static content is done